server/libs/userlibs.js

```diff
@@ -72,30 +89,26 @@ const logIn = function(body) {
                 }
             })
         }).catch((error) => {
-            reject({
-                status : statusCodes.badRequest,
-                message : `Something ${messages.notMatch}`,
-                error
-            });
+            reject(error);
```

Comment:
no status of error and message and not error: error why?

Reply:
Because error already is an object and has status and message in it